modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/shared/CodeMirrorEditor.js

```diff
@@ -671,6 +674,27 @@ const CodeMirrorEditor = React.forwardRef(
 		const editorRef = useCombinedRefs(ref, innerRef);
 		const {availableLanguages} = useContext(ThemeContext);
 
+		const [foldLoading] = useAsyncCall(
+			() => {
+				if (folded && editorRef.current) {
+					editorRef.current.operation(() => {
+						for (
+							let line = editorRef.current.firstLine() + 1;
+							line <= editorRef.current.lastLine() - 1;
+							++line
+						) {
+							editorRef.current.foldCode(
+								{ch: 0, line},
+								null,
+								'fold'
+							);
+						}
+					});
+				}
+			},
+			folded ? 200 : 0
```

Comment:
@oliv-yu Looking at this now, I'd say it's still a good idea to have a `foldInitializationDelay` prop since the `200` delay should only be for code mirror editors inside of modals to wait for the opening animation. `foldInitializationDelay` can be default `0` and then for "View Raw Response" editor you can set `foldInitializationDelay={200}`

Otherwise if future code mirror editors have `fold=true`, there could be an unnecessary delay.

Reply:
Ok! Made the change with `foldInitializationDelay`. I actually squashed the commit, let me know if there's any confusion 😅